cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -90,6 +90,12 @@ func LogOrWriteConfig(logger klog.Logger, fileName string, cfg *config.KubeSched
 		return err
 	}
 
+	defaultPluginString := "Plugins Enabled By Default:"
+	for _, v := range configv1.GetDefaultPlugins.MultiPoint.Enabled {
```

Comment:
#109285 argues that we cannot see which plugins are enabled at the end.

> Unfortunately, the printed config does not mention which plugins are enabled besides those which are explicitly mentioned in the configuration (through enabled and disabled fields).

> I am reporting the issue in order start a discussion on how to better let a user know which plugins are enabled by default. 

The default plugins might be disabled by the configuration. Also, users can do like "disable defaultpluginA only in the score extension point".
I don't see those being addressed in this PR.

Reply:
Also, note that we don't necessarily need to add this functionality in this existing config log. 
Probably, it's easier to implement in the scheduler (/pkg/scheduler) not here.

Reply:
Would it be feasible if the logged component config displays all the enabled or disabled plugins for each extension point.

Reply:
Logging plugins which are **actually** enabled in each extension point would be useful (+ I think that's what the original issue wants) and it's feasible to be implemented in the scheduler(under /pkg/scheduler), not here.

Reply:
https://github.com/kubernetes/kubernetes/blob/c13d3369d588bf59eec8812327e421f1ce47c87a/pkg/scheduler/framework/runtime/framework.go#L296

FYI, the scheduler initializes all necessary plugins around here

Reply:
@sanposhiho i have made the necessary changes can u please review it